File: uri 1160 - Population Increase.py
n = int(input())
for i in range(n):
    pa, pb, g1, g2 = map(float, input().split(" "))
    pa, pb = int(pa), int(pb)
    y = 0
    while pa <= pb:
        pa += int((pa * g1) /100)
        pb += int((pb * g2) /100)
        y += 1
        if y > 100:
            print("Mais de 1 seculo.")
            break
    if y <= 100:
        print(f"{y} anos.")

File: uri 1160 - Population Increase.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int T,PA,PB,Y;
    double GA,GB;
    scanf("%d",&T);
    while(T>0){
        Y=0;
        scanf("%d %d %lf %lf",&PA,&PB,&GA,&GB);
        while(PA<=PB){
            Y++;
            PA=PA +(int)((GA*(double)PA)/100.00);
            PB=PB +(int)((GB*(double)PB)/100.00);
            if(Y>100){
                printf("Mais de 1 seculo.\n");
                break;
            }

        }
        if(Y<=100)
            printf("%d anos.\n",Y);
        T--;
    }

    return 0;
}
